conf/modules.config

```diff
@@ -998,10 +997,11 @@ process{
     }
 
     withName: ".*:ANNOTATION_MERGE:ENSEMBLVEP" {
-        ext.prefix       = {"${meta.id}_snpEff"}
```

Comment:
@maxulysse this seems weird. do you recall why adding a snpeff prefix to a VEP process?

Reply:
yes, simply because this vcf is coming from VEP, and if we don't add this snpeff prefix, we will have a collision name as the full name should be with the snpeff prefix, the vep prefix is already in the module if I remember correctly

Reply:
ok i don't see it, and we do add the VEp thing when running VEP alone

```
 vep \\
        -i $vcf \\
        -o ${prefix}.ann.vcf \\
        $args \\
        --assembly $genome \\
        --species $species \\
        --cache \\
        --cache_version $cache_version \\
        --dir_cache $dir_cache \\
        --fork $task.cpus \\
        --vcf \\
        --stats_file ${prefix}.summary.html
        ```

Reply:
Might need to update this config then... it could be that I remove the _VEP part from the module and forgot to update to _snpEff_VEP